docs/admin/kubeadm-token.md

```diff
@@ -0,0 +1,46 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm token
+notitle: true
+---
+{% capture overview %}
+# Kubeadm token
+Bootstrap Tokens are used for establishing bidirectional trust between a node joining 
+the cluster and a the master node, as described in [Authenticating with Bootstrap Tokens](bootstrap-tokens.md).
+
+`kubeadm-init` creates an initial token with 24h TTL; following commands allow to manage 
+such token and also to create and manage new ones.
+
+But, what is a Bootstrap Token more exactly?
+ 
+ - It is a string the form `[a-z0-9]{6}.[a-z0-9]{16}"`; the former part is the 
+   public token ID, and the latter is the Token Secret, which must be kept private at all circumstances.
```

Comment:
Global comment: Avoid capitalizing things that aren't proper names. For example, I don't think we should capitalize Bootstrap Token or Token Secret.